src/opera/pge/disp_s1/templates/disp_s1_measured_parameters.yaml

```diff
@@ -166,3 +278,119 @@ metadata/dolphin_software_version:
   attribute_type: processingInformation
   description: Version of the Dolphin software used to generate the product
   display_name: DolphinSoftwareVersion
+
+metadata/pge_runconfig:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: The full PGE runconfig YAML file used to generate the product.
+  display_name: PGERunConfig
+metadata/algorithm_parameters_yaml:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: The full algorithm parameters YAML file used to generate the product.
+  display_name: AlgorithmParametersYAML
+metadata/dolphin_workflow_config:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: The configuration parameters used by `dolphin` during the processing.
+  display_name: DolphinWorkflowConfig
```

Comment:
I worry a bit about including these wholesale within the ISO xml. I could see it introducing subtle XML parsing errors based on what is in the configs we include, so I think better to just leave this section out entirely.